src/Tribe/Rewrite.php

```diff
@@ -509,4 +510,51 @@ protected function get_dynamic_matchers( array $query_vars ) {
 	protected function get_post_types() {
 		return [ 'tribe_events', 'tribe_venue', 'tribe_organizer' ];
 	}
+
+	/**
+	 * Overrides the base method, from commmon, to filter the parsed query variables and handle some cases related to
+	 * the `eventDisplay` query variable.
+	 *
+	 * {@inheritDoc}
+	 */
+	public function parse_request( $url, array $extra_query_vars = [], $force = false ) {
+		if ( ! has_filter( 'tribe_rewrite_parse_query_vars', [ $this, 'filter_rewrite_parse_query_vars' ] ) ) {
+			add_filter( 'tribe_rewrite_parse_query_vars', [ $this, 'filter_rewrite_parse_query_vars' ], 10, 3 );
+		}
+
+		return parent::parse_request( $url, $extra_query_vars, $force ); // TODO: Change the autogenerated stub
```

Comment:
:warning: Comment refers to a TODO task "Change the autogenerated stub"

<sub>hash: 0e8aead55201aecf0041b770dce4f542</sub>